scripts/quests/jeuno/LB10_Beyond_Infinity.lua

```diff
@@ -0,0 +1,232 @@
+-----------------------------------
+-- Beyond Infinity
+-----------------------------------
+-- Log ID: 3, Quest ID: 137
+-- Nomad Moogle : !pos 10.012 1.453 121.883 243
+-----------------------------------
+require('scripts/settings/main')
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ruludeID = require('scripts/zones/RuLude_Gardens/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.BEYOND_INFINITY)
+
+quest.reward =
+{
+    fame = 50,
+    fameArea = JEUNO,
+    title = xi.title.BUSHIN_ASPIRANT,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    -- Note: This step only happens with one, very specific option in the previous quest.
+    -- In most cases, the quest will already be accepted.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getQuestStatus(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.PRELUDE_TO_PUISSANCE) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Nomad_Moogle'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(10045, 0, 1, 5, 0, 1)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10045] = function(player, csid, option, npc)
+                    -- This options start quest.
+                    if option == 13 or option == 14 or option == 19 or option == 20 or option == 21 then
```

Comment:
Is there a warp option for 13?  Or does it just stay in Jeuno?

Reply:
you stay